packages/@react-spectrum/combobox/src/MobileComboBox.tsx

```diff
@@ -421,7 +463,9 @@ function ComboBoxTray(props: ComboBoxTrayProps) {
             )
           }
           ref={listBoxRef}
-          onScroll={onScroll} />
+          onScroll={onScroll}
+          onLoadMore={onLoadMore}
+          isLoading={isLoading} />
```

Comment:
Added `isLoading` here so the client side filtering story would continue to fetch items if there weren't enough to fill the tray (if it doesn't fill the tray, then the user can't scroll to trigger loadMore)

Trying to figure out a better way to do this so that the scroll wheel only shows up when the user actually triggers a scroll event